draft/spec/index.html

```diff
@@ -184,19 +184,19 @@ <h2>Need</h2>
            digital repositories.
         </p>
         <p>
-           A general observation is that the contents of a digital repository -- that is, the digital files and metadata
-           that an institution might wish to manage -- are largely stable. Once content has been accessioned, it is
-           unlikely to change significantly over its lifetime. This is in contrast to the software applications that
-           manage these contents, which are ephemeral, requiring constant updating and replacement. Thus, transitions
-           between application-specific methods of file management to support software upgrades and replacement cycles
-           can be seen as unnecessary and risky change which affects the long-term stable objects in support of the
-           short-term, ephemeral software.
+           A general observation is that the contents of a digital repository &#8212; that is, the digital files and
+           metadata that an institution might wish to manage &#8212; are largely stable. Once content has been
+           accessioned, it is unlikely to change significantly over its lifetime. This is in contrast to the software
+           applications that manage these contents, which are ephemeral, requiring constant updating and replacement.
+           Thus, transitions between application-specific methods of file management to support software upgrades and
+           replacement cycles can be seen as unnecessary and risky change which affects the long-term stable objects
+           in support of the short-term, ephemeral software.
         </p>
         <p>
-           By providing a specification for the file and directory layout on disk, OCFL is an attempt at reducing, or
-           even eliminating, the need for these transitions. As an application-independent specification, conforming
-           applications will natively "understand" the underlying file structure without needing to first transition
-           these contents to their own format.
+           By providing a specification for the file and directory layout on disk or in an object store, OCFL is an
```

Comment:
I prefer the active voice but instead of expanding the scope of this PR I'll leave as-is for now. Simply changing to "attempts" sounds awkward to me because OCFL has insufficient agency to attempt anything itself. I think we can find a solution for both issues but it requires a larger edit